Sources/Fluent/Entity/Entity.swift

```diff
@@ -19,6 +19,15 @@ public protocol Entity: Preparation, NodeConvertible {
         like pivots.
     */
     static var name: String { get }
+    
+    /**
+        The name of the column that corresponds
+        to this entity's key.
+     
+        Return nil to use the database's global
+        idKey.
+     */
+    static var idKey: String? { get }
```

Comment:
Trying to re-implement I now seem to remember why: If let non-nil (with a default `database?.driver.idKey ?? "id"`), if the entity is used in another database without changing the static `database` variable, it'd reference the first database's idKey value in the second database instead, so the behavior would change in that case.